contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.2.0--2.3.0.sql

```diff
@@ -3048,6 +3048,9 @@ AS 'babelfishpg_tsql', 'update_user_catalog_for_guest';
  
 CALL sys.babelfish_update_user_catalog_for_guest();
 
+-- Drop this procedure after it is executed once during upgrade.
```

Comment:
Instead of this comment here we should rather add a comment before line no. 3045 to specify that it is a temporary procedure which serves XYZ functionality.